bin/vyper

```diff
@@ -44,8 +44,8 @@ def get_asm(asm_list):
 if __name__ == '__main__':
 
     output_format = {}
-    output_format['abi'] = lambda code: compiler.mk_full_signature(code)
-    output_format['json'] = lambda code: json.dumps(compiler.mk_full_signature(code))
+    output_format['python_abi'] = lambda code: compiler.mk_full_signature(code)
+    output_format['abi'] = lambda code: json.dumps(compiler.mk_full_signature(code))
     output_format['bytecode'] = lambda code: '0x' + compiler.compile(code).hex()
     output_format['bytecode_runtime'] = lambda code: '0x' + compiler.compile(code, bytecode_runtime=True).hex()
```

Comment:
by consistency with `bytecode_runtime`, a better name may be `abi_python` - WDYT?

Reply:
Sure I agree, however we must leave the old `json`. I know for example tools like embark rely on the using json and this will be then an incompatible change.

Reply:
Sure! Just note that changing the behaviour of `-f abi` will be a slightly incompatible change as well (depending on the syntax-tolerance of whatever parses the resulting ABI).